backend/src/org/jetbrains/kotlinconf/backend/Main.kt

```diff
@@ -16,9 +16,13 @@ import io.ktor.websocket.*
 val gson = GsonBuilder().setPrettyPrinting().serializeNulls().create()
 
 fun Application.main() {
-    val config = environment.config.config("service")
-    val mode = config.property("environment").getString()
+    val config = environment.config
+    val serviceConfig = config.config("service")
+    val mode = serviceConfig.property("environment").getString()
     log.info("Environment: $mode")
+    val sessionizeConfig = config.config("sessionize")
+    val sessionizeUrl = sessionizeConfig.property("url").getString()
+    val sessionizeInterval = sessionizeConfig.property("interval").getString().toLong()
```

Comment:
Nitpick - What do you think about creating extension function to directly get int?
```kotlin
fun ApplicationConfigValue.getLong() = getString().toLong()
```

or maybe even going a step further, we could create extensions to remove the `property("").getString()` calls?
```kotlin
fun ApplicationConfig.getPropertyString(path: String) = property(path).getString()
```

Reply:
Extension function just to replace one usage that contains single expression with two function calls... it does not sound like a good idea for me. Why designing your own API if the current one is nearly as good.